### PR TITLE
Add telemetry page

### PR DESCRIPTION
**Pull request by avestuk**
I need input from @domodwyer about the CLI and API methods of disabling telemetry. 

The general language of the article can be reviewed though.

### CODE REVIEW

File: _docs/reference/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# StorageOS Telemetry
+
+StorageOS collects telemetry from StorageOS clusters via two different methods
+for two different purposes.
+
+## Sentry.io
+
+StorageOS sends crash dumps and information about the StorageOS cluster to sentry.io.
+This information helps our devleopers monitor and fix crashes. This information
```
Comment (actionbuddha) [inline _docs/reference/telemetry.md]: developers

File: _docs/reference/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Sentry.io
+
+StorageOS sends crash dumps and information about the StorageOS cluster to sentry.io.
+This information helps our devleopers monitor and fix crashes. This information 
+includes the cluster version, node counts and license information as well as
+any crash dumps.
+
+Information is sent to Sentry.io once per day or when a process inside the
+StorageOS container crashes. All StorageOS clusters with a routable connection
+to the internet will send crash dumps to Sentry using an HTTPS connection.
```
Comment (actionbuddha) [inline _docs/reference/telemetry.md]: 'over tcp/443' - just to be explicit

Comment (actionbuddha) [reply _docs/reference/telemetry.md]: Also - will we honour env variable https_proxy? Worth finding out if Dom knows.

Comment (avestuk) [reply _docs/reference/telemetry.md]: @domodwyer Do you know if we honour that?

Comment (avestuk) [reply _docs/reference/telemetry.md]: @actionbuddha We do respect env variables. Please see the latest commit for more information.

File: _docs/reference/telemetry.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+# StorageOS Telemetry
+
+StorageOS collects telemetry and error reports  from StorageOS clusters via two
+different methods for two different purposes.
+
+## Sentry.io
+
+StorageOS sends crash dumps and information about the StorageOS cluster to [sentry.io]
+(https://sentry.io). This information helps our developers monitor and fix crashes.
```
Comment (Arau) [inline _docs/reference/telemetry.md]: Broken link when rendered. The jump line breaks it

File: _docs/reference/telemetry.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+You can use the following environmental variables to disable or enable telemetry.
+
+```bash
+DISABLE_TELEMETRY       # Disable the DNS query and once per day Sentry.io
+reporting
+DISABLE_ERROR_REPORTING # Disable SentryIO crash dump reports
+```
+
+You can find more information about StorageOS environment variables
+[here]({%link _docs/reference/envvars.md %})
```
Comment (Arau) [inline _docs/reference/telemetry.md]: missing final '.'

File: _docs/reference/telemetry.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+StorageOS collects telemetry and error reports  from StorageOS clusters via two
+different methods for two different purposes.
+
+## Sentry.io
+
+StorageOS sends crash dumps and information about the StorageOS cluster to
+[sentry.io](https://sentry.io). This information helps our developers monitor
+and fix crashes.
+
+Information is sent to Sentry.io once per day or when a process inside the
```
Comment (actionbuddha) [inline _docs/reference/telemetry.md]: Inconsistent caps here - Exclusive of the heading, do we want Sentry.io or sentry.io?

**Comment by domodwyer** [comment]
I can't submit my review as it's merged - @actionbuddha has the relevant comments though.

**Pull request merged**